spec/main_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'spec_helper'
+require_relative '../validate_service.rb'
+require 'json_spec' # Require Gem file for testing JSON..?
+require 'json' #require for use of .to_json ..?
+
+# Create ValidateService for testing purposes comprised of the interface we are testing.
+class ValidateService < Hoodoo::Services::Service
+  comprised_of ValidateInterface
+end
+
+describe "validate service" do
+  include Rack::Test::Methods
+  # Create rack builder object for the test.
+  context "get /1/Hello" do
+    let(:app) {
+      Rack::Builder.new do
+        use( Hoodoo::Services::Middleware )
+        run( ValidateService.new )
+      end
+    }
+
+    it "Passes if http status code == 200" do
```

Comment:
It's helpful if you can make these read more like a sentence eg:
```suggestion
    it "returns http status code == 200, when the payload is valid" do
```